vms/platformvm/genesis/genesistest/genesis.go

```diff
@@ -69,7 +69,9 @@ type Config struct {
 
 func New(t testing.TB, c Config) *platformvmgenesis.Genesis {
 	if len(c.NodeIDs) == 0 {
-		c.NodeIDs = DefaultNodeIDs
+		for _, shortNodeID := range DefaultNodeIDs {
+			c.NodeIDs = append(c.NodeIDs, shortNodeID.NodeID())
+		}
```

Comment:
I don't think we should be transforming the types here. I think the users of this util should be specifying the `NodeID` type they expect to have used.